### PR TITLE
Fixed tests

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: ahcorde <ahcorde@gmail.com>

# 🦟 Bug fix

## Summary

According with the official [Qt documentation](https://doc.qt.io/qt-5/qguiapplication.html#QGuiApplication).

```
Warning: The data referred to by argc and argv must stay valid for the entire lifetime of the QGuiApplication object.
In addition, argc must be greater than zero and argv must contain at least one valid character string.
```

These changes should fix the segmentation faults

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

**Review by mjcarroll — COMMENTED** [review]
Just one nit/question

**Comment by chapulina** [comment]
> These changes should fix the segmentation faults

What segmentation faults? Can you provide more context?

**Comment by ahcorde** [comment]
> > These changes should fix the segmentation faults
> 
> What segmentation faults? Can you provide more context?

In general I can see many times `UNIT_Application_TEST`, `UNIT_WorldControl_TEST` and UNIT_WorkdStats_TEST` failing because some seg faults. For example: https://github.com/ignitionrobotics/ign-gui/pull/221/checks?check_run_id=2931467916

By the way I ran `ASAN` inside a docker file and this seg faults always happen. it's not fully solved but it should remove some of them

File: src/Plugin.cc [hunk 1]
```diff
@@ -91,7 +91,8 @@ Plugin::Plugin() : dataPtr(new PluginPrivate)
 /////////////////////////////////////////////////
 Plugin::~Plugin()
 {
-  delete this->dataPtr->pluginItem;
+  if (this->dataPtr->pluginItem)
```
[the remark this answers is not in the document]
Comment (chapulina) [reply src/Plugin.cc:94]: I'm also curious about why this was added. But it shouldn't hurt, so I'm ok leaving it in.

**Review by chapulina — APPROVED** [review]
Thanks for the context, LGTM. I pushed some indentation fixes in 1b8f1c0a65343732682bcf3272e91e777c6de47f.

**Pull request merged**